openshift/resultsdb-updater-test-template.yaml

```diff
@@ -64,7 +64,10 @@ objects:
         'stomp_ssl_key': '/etc/resultsdb/umb-consumer.key.pem',
         'resultsdb-updater.log_level': logging.DEBUG,
         'resultsdb-updater.resultsdb_api_url': '${RESULTSDB_API_URL}',
-        'resultsdb-updater.resultsdb_api_ca': None,
+        # TODO make certificate verification work
```

Comment:
As long as you don't use the Python requests from pip you won't need this set as the system CA bundle should work just fine unless the CA is not installed of course.

Reply:
The CA is not installed.